definitions/npm/big.js_v3.x.x/test_big.js_v3.x.x.js

```diff
@@ -1,6 +1,5 @@
 // @flow
-
-import Big from "big.js";
+import Big from 'big.js';
```

Comment:
Guessing this wasn't intentional?

(Since libdefs are signed/versions once they are installed with the CLI, it's good not to change the contents unless there's an update/improvement/fix/etc)